src/mono/wasm/runtime/startup.ts

```diff
@@ -311,7 +312,7 @@ function mono_wasm_pre_init_essential(isWorker: boolean): void {
     if (!isWorker)
         Module.addRunDependency("mono_wasm_pre_init_essential");
 
-    if (runtimeHelpers.diagnosticTracing) console.debug("MONO_WASM: mono_wasm_pre_init_essential");
+    if (runtimeHelpers.diagnosticTracing || isWorker) console.debug("MONO_WASM: mono_wasm_pre_init_essential");
```

Comment:
let's not make it more noisy please. 
Long term I would prefer all behind `if (runtimeHelpers.diagnosticTracing)` even on workers.
I guess for now it's ok

Reply:
@pavelsavara yea I'll back this out once the PR is out of draft.  I didn't have time to revert the noise before I had to stop for the day.

But also note that some of this code (before the worker receives the MonoConfig) can't reliably use `runtimeHelpers.diagnosticTracing`.  I thing we can probably leave it quiet in Release builds, but I'd like to have something in Debug for resolving these kinds of startup problems.

Reply:
Reverted